{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "f31d2d6d_a6b5cf7b",
        "filename": "project.py",
        "patchSetId": 5
      },
      "lineNbr": 3583,
      "author": {
        "id": 1096638
      },
      "writtenOn": "2024-12-18T18:53:34Z",
      "side": 1,
      "message": "While unlikely, a user could have `subproject-objects` in their repo checkout path. Let\u0027s handle that case.",
      "revId": "47cc1e0863050921b43c5a4ea06afd758bed6a3b",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    }
  ]
}